assets/css/combined.css

```
@@ -1,4 +1,930 @@
+    transition: 0.5s;
+    bottom: -8px;
+    height: 1.5px;
+  }
+  .drawer-nav {
+    background-color: #fff;
+    position: fixed;
+    width: 100%;
+    height: 98%;
+    /* IE8以下とAndroid4.3以下用フォールバック */
+    height: -webkit-calc(100% - 80px);
+    height: calc(100% - 80px);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)